core/model/listeners.py

```diff
@@ -108,8 +108,6 @@ def _site_configuration_has_changed(_db, cooldown=1):
 # catch most that slip through the cracks.
 @event.listens_for(Collection.children, "append")
 @event.listens_for(Collection.children, "remove")
-@event.listens_for(Collection.libraries, "append")
-@event.listens_for(Collection.libraries, "remove")
```

Comment:
This is now handled by calling the `site_configuration_changed` function from the controller explicitly instead of using this listener.